components/create/Review.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import Chip from '@mui/material/Chip';
+import Stack from '@mui/material/Stack';
+import CreatePortfolioContext from 'components/create/CreatePorfolioContext';
+import { Typography } from '@mui/material';
+import { useState, useContext, useRef, RefObject } from 'react';
+import { Divider } from '@mui/material';
+
+const Review = () => {
+  const { title, name, headshot, about, work, skills, projects, contact } =
+    useContext(CreatePortfolioContext);
+  return (
+    <div>
```

Comment:
all this could be copy/pasted into a common view component that could be used in both Review and the View page

Reply:
Good idea.